web/api/mastermind/order.js

```diff
@@ -0,0 +1,72 @@
+const Joi = require('joi');
+const auth = require('../../lib/authentications');
+const validation = require('../validations/orders');
+const orderHandlers = require('../../lib/handlers/order');
+
+const basePath = '/api/3.0';
+const { applications } = validation.definitions;
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${basePath}/events/{eventId}/orders/{orderId}`,
```

Comment:
May I request to remove the {orderId} ? As discussed yesterday, there should be only 1 order per user per event, so that load, if not done in the context of a dojo-champion, should only return the user's order

Reply:
I'm a bit unsure how exactly we should handle this one. `/events/{eventId}/orders` should return all orders for an event that you're allowed to see. You may be a ticketing admin, but still book a ticket for your child, so one endpoint doesn't work.

Part of my wants to say `/events/{eventId}/my-order` to go along with it, but I'm not a big of that. Other options:

Reply:
Yet the code behind makes use of the current userId to list those orders.
Either we keep it and it becomes the list of orders that the user can see for this event (which becomes problematic if he's himself booked as well) or 
We use the different endpoint, in which case I'd be happy we remove the "new" one as we have to look into permissions (that I don't want to go into now)
I believe the best one is probably `/events/{eventId}/just-give-me-my-orders-please`, but i'd prefer to avoid "prefixing" the api with `my` (considering the way our routes are being created). 
Another solution would be to reverse the dependency then, like you did for `/users/{userId}/events/{eventId}/orders` but i'd be more inclined to reduce it to `/users/{userId}/orders?eventId=`

Reply:
Actually, yeah, I like the idea of `/users/{userId}/orders?eventId={eventId}` the most because then you can get all orders for a user by just omitting the eventId, which could be useful for a future version of my tickets.

Reply:
I also like the idea of allowing the URL to be `/users/me/orders` so you don't have to bother getting the logged in user's ID ;)

Reply:
What if `me` is a valid GUID ? :taco:

Reply:
@heyitsmaimai Url of `events/{eventId}/order/{orderId}` to be changed to `users/{userId}/orders` with optional query param `eventId`

Reply:
Actually, I assume it should really be `?query[eventId]=asdf`?